File: UF2/pyBiblioteca/libro.py
class Libro:
    def __init__(self, titulo, autor, cantidad) -> None:
        self.titulo: str = titulo
        self.autor: str = autor
        self.cantidad: int = cantidad
    
    def informacion(self):
        print(f"Titulo: {self.titulo}, Autor: {self.autor}, Cantidad: {self.cantidad}")
    
    def prestar(self):
        if (self.cantidad > 0):
            self.cantidad -= 1
            print(f"Libro prestado: {self.titulo}, autor: {self.autor}")
        else:
            print(f"No hay ejemplares disponibles de {self.libro}")
    
    def devolver(self):
        self.cantidad += 1
        print(f"Libro prestado: {self.titulo}, autor: {self.autor}")

File: UF2/ktBiblioteca/libro.kt
package Biblioteca

class Libro(val titulo: String, val autor: String, var cantidad: Int) {

    fun prestar() {
        if (cantidad > 0) {
            cantidad--
            println("Libro prestado: $titulo, Autor: $autor")
        } else {
            println("No hay ejemplares disponibles de $titulo")
        }
    }

    fun devolver() {
        cantidad++
        println("Libro devuelto: $titulo, Autor: $autor")
    }

    fun informacion() {
        println("Título: $titulo, Autor: $autor, Ejemplares Disponibles: $cantidad")
    }
}